content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,185 @@
+Metapackage would only include stable instrumentations (i.e., those which are at
+v1.0 or greater). For consistency, we propose a single project-wide way to
+enable unstable behavior for end-users. The goal of this proposal is to reduce
+the likelihood of end-user adoption of unstable packages, components, config, or
+API surface area without an explicit opt-in to this behavior.
+
+We do not make this change lightly, and it wasn’t feasible to make it sooner –
+end-user feedback has been a crucial aspect of our development process, and
+early adoption of unstable instrumentation and features has led to better
+outcomes overall. However, given the scope of OpenTelemetry, it can be difficult
+for users to evaluate individual component stability when installing
+dependencies. By moving to a ‘stable by default’ approach, we can ensure that
+users will not be surprised by unexpected changes.
+
+## Adjusting stability requirements for components
+
+Second, we are adjusting our overall posture when it comes to instrumentation
+and instrumentation stability. Our goal, as a project, has always been to see
+other libraries, frameworks, runtimes, and tools
+[natively adopt OpenTelemetry](https://www.youtube.com/watch?v=l8xiNOCIdLY).
+This is still the goal – we want other developers to embed our APIs into their
+software, rather than requiring you to download and install libraries that
+inject instrumentation code into third party dependencies. However, many users
+have come to see our ‘contrib’ repositories and distributions as generic
+replacements for system telemetry agents.
+
+We’re thrilled that we’ve had so many users adopt the Collector as a
+vendor-agnostic solution for collecting telemetry data, and we believe that it’s
+a strong testament to the long-term viability of the project. What has become
+challenging is that our contrib repositories – especially in the Collector –
+represent a huge amount of work for maintainers and a constant drag on project
+velocity. Even in a world where we had twice as many contributors and
+maintainers, it would still be a challenge to accept all desired receivers or
+instrumentation libraries and ensure they all met a sufficiently high quality
+bar for release.
+
+Our next proposal is designed to address this tension by more tightly scoping
+what instrumentation libraries and receivers we’ll consider for inclusion and
+stability. In short, instrumentation should be an implementation of a semantic
+convention. This has quite a few implications, some of which include:
+
+- Existing instrumentation libraries or Collector receivers that do not follow
+  an established semantic convention will be treated as experimental/unstable
+  until such time those conventions exist.
+- SIGs should no longer accept instrumentation libraries or receivers that do
+  not implement a semantic convention.
+- The priority for instrumentations should be driven by semantic convention
```

Comment:
For focusing the community and making it clear what we want to drive I like this.

I"m worries this will put pressure to start up N semantic convention groups that don't exist today.  We should make sure our Project onboarding process and traige can handle this new pressure.

cc @open-telemetry/specs-semconv-maintainers